decidim-core/app/assets/stylesheets/decidim/layouts/_view.scss

```diff
@@ -12,6 +12,11 @@
   }
 }
 
+.icon-arrow-thin-right {
```

Comment:
Do we want this class to be used in the whole of decidim or will it only be used in the cards used by debates? If so maybe it should go in another scss file related to cards or to debates.